Pipeline_example/scripts/cp_parallel.py

```diff
@@ -0,0 +1,136 @@
+
+
+"""
+Taken from Jenna's script from the Cellpainting repo
```

Comment:
Great string description of the script. Consider adding the link to the `cp_paralell.py` file

Reply:
I agree with this recommendation! 😄

Reply:
Got it, thanks!